File: 02_Estructuras de Control/02_Condicionales/04_Multiples/Python/ejemplo2.py
# Programa: Mostrar mes y número de días

print("===== Mostrar mes y número de días =====\n")

# Entrada
numero = int(input("Ingrese un número del 1 al 12: "))

# Proceso + Salida
if numero == 1:
    print("Enero - 31 días")
elif numero == 2:
    print("Febrero - 28 días (29 en bisiesto)")
elif numero == 3:
    print("Marzo - 31 días")
elif numero == 4:
    print("Abril - 30 días")
elif numero == 5:
    print("Mayo - 31 días")
elif numero == 6:
    print("Junio - 30 días")
elif numero == 7:
    print("Julio - 31 días")
elif numero == 8:
    print("Agosto - 31 días")
elif numero == 9:
    print("Septiembre - 30 días")
elif numero == 10:
    print("Octubre - 31 días")
elif numero == 11:
    print("Noviembre - 30 días")
elif numero == 12:
    print("Diciembre - 31 días")
else:
    print("Número fuera de rango (1-12)")

input("\nPresione Enter para salir...")

File: 02_Estructuras de Control/02_Condicionales/04_Multiples/CSharp/ejemplo2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejercicio1Multiples
{
    internal class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("===== Mostrar mes y número de días =====");
            Console.WriteLine();

            // Entrada

            Console.Write("Ingrese un número del 1 al 12: ");
            int numero = int.Parse(Console.ReadLine());

            // Proceso + Salida

            switch (numero)
            {
                case 1:
                    Console.WriteLine("Enero - 31 días");
                    break;
                case 2:
                    Console.WriteLine("Febrero - 28 días (29 en bisiesto)");
                    break;
                case 3:
                    Console.WriteLine("Marzo - 31 días");
                    break;
                case 4:
                    Console.WriteLine("Abril - 30 días");
                    break;
                case 5:
                    Console.WriteLine("Mayo - 31 días");
                    break;
                case 6:
                    Console.WriteLine("Junio - 30 días");
                    break;
                case 7:
                    Console.WriteLine("Julio - 31 días");
                    break;
                case 8:
                    Console.WriteLine("Agosto - 31 días");
                    break;
                case 9:
                    Console.WriteLine("Septiembre - 30 días");
                    break;
                case 10:
                    Console.WriteLine("Octubre - 31 días");
                    break;
                case 11:
                    Console.WriteLine("Noviembre - 30 días");
                    break;
                case 12:
                    Console.WriteLine("Diciembre - 31 días");
                    break;
                default:
                    Console.WriteLine("Número fuera de rango (1-12)");
                    break;
            }

            Console.WriteLine("\nPresione cualquier tecla para salir...");
            Console.ReadKey();
        }
    }
}